client/extensions/woocommerce/app/products/product-form-variations-table.js

```diff
@@ -8,12 +8,12 @@ import { find, isNumber } from 'lodash';
 /**
  * Internal dependencies
  */
+import CompactFormToggle from 'components/forms/form-toggle/compact';
```

Comment:
I moved the line from a few lines below because it was the only one not alphabetized.